README.md

```diff
@@ -94,6 +94,30 @@ r.word_of_the_day()
     # Output: {"word": "qualtagh", "definations": [{"text": "The first person one encounters, either after leaving one\'s home or (sometimes) outside one\'s home, especially on New Year\'s Day.", "source": "wiktionary", "partOfSpeech": "noun"}, {"text": "A Christmas or New Year\'s ceremony, in the Isle of Man; one who takes part in the ceremony. See the first extract.", "source": "century", "partOfSpeech": "noun"}]}
     ```
 
+## Development
+
+ Assuming that you have [`Python`](https://www.python.org/downloads/release/python-370/) and [`pipenv`](https://docs.pipenv.org) installed, set up your environment and install the required dependencies like this instead of the `pip install random-word` defined above:
+
+    ```sh
```

Comment:
Please remove indentation from all the code snippets

Reply:
If I do that It becomes all messy, I have done that in the previous commit but it became so messy.
Still, I'll do that.